h/models/annotation_slim.py

```diff
@@ -31,9 +31,7 @@ class AnnotationSlim(Base):
     )
     """The value of annotation.id, named here pubid following the convention of group.pubid"""
 
-    annotation = sa.orm.relationship(
```

Comment:
backref is considered legacy on SQLA 2.0:

https://docs.sqlalchemy.org/en/20/orm/backref.html